commands/deploy.go

```diff
@@ -1,20 +1,74 @@
 package commands
 
 import (
+	"bytes"
 	"fmt"
+	"net/http"
 	"os"
+	"path/filepath"
+	"strings"
+	"time"
 )
 
 // DeployCmd handles deploying an app to Section
-type DeployCmd struct{}
+type DeployCmd struct {
+	URL string
+}
 
 // Run deploys an app to Section's edge
 func (a *DeployCmd) Run() (err error) {
-	path, err := os.Getwd()
+	// path, err := os.Getwd()
+	// if err != nil {
+	// 	panic(err)
+	// }
+	path := "./"
+	var files []string
+	err = filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+		if info.IsDir() {
+			return nil
+		}
+		if strings.Contains(path, ".git/") {
+			return nil
+		}
+		if strings.Contains(path, ".lint/") {
```

Comment:
Might be more general purpose to have an optional exclude pattern which defaults to anything beginning with a dot `.` by default.

Reply:
I am planning to add `.section.json` config file or maybe a `.section/config.json` and `\.*+` would match that perfectly.

Reply:
What would such a json file contain?

Reply:
The environment ID. What if the user has multiple projects for multiple envs? How do we know which one we are deploying?

Reply:
@manibatra just told me that I only needed to use account ID and not env ID. So I will not add a config file per project for now.